tasks/message/message.go

```diff
@@ -3,6 +3,7 @@ package message
 import (
```

Comment:
did you mean to land this in the same PR?

Reply:
i did not. this is from #165. I can rebase to remove that commit